go.mod

```diff
@@ -83,6 +83,7 @@ require (
 	go.uber.org/zap v1.24.0
 	go.viam.com/api v0.1.80
 	go.viam.com/dynamixel v0.0.0-20210507131419-60a9033552cb
+	go.viam.com/slam v0.1.22
```

Comment:
@jeremyrhyde @JohnN193 @kkufieta @nicksanford do you know why this has shown up on my pr and should I be concerned? Only running the make prior to committing.

Reply:
This line is already on go.mod: https://github.com/viamrobotics/rdk/blob/main/go.mod#L86

Does it go away when you rebase?

Reply:
Does thanks.